lib/bundle/cask_dumper.rb

```diff
@@ -5,11 +5,7 @@ def self.reset!
     end
 
     def self.casks
-      @casks ||= if Bundle.cask_installed?
-        `brew cask list -1 2>/dev/null`.split("\n").map { |cask| cask.chomp " (!)" }
-      else
-        []
-      end
+      @casks ||= `brew cask list -1 2>/dev/null`.split("\n").map { |cask| cask.chomp " (!)" }
```

Comment:
`Bundle.cask_installed?` is still required. Otherwise `brew bundle check` will automatically install cask, which would be unexpected.